src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys-sphinx-theme-variable.css

```diff
@@ -134,10 +142,32 @@ html[data-theme="light"] {
   --pst-color-code-s1: #b35000;
   --pst-color-code-c1: #095d0a;
   /**
-* sphinx design primary color
+* sphinx design
 */
   --sd-color-primary: var(--pst-color-text-base);
+  --ast-color-enable-card-background: #ffffff;
+  --ast-color-enable-border: #d9d9d9;
+  --ast-color-hover-card-background: #ececec;
+  --ast-color-hover-border: #8e8e8e;
 
+  --ast-color-sphinx-design-primary: #353535;
+  --ast-color-sphinx-design-background: #ffffff;
+  --sd-color-primary: #141414;
+  --sd-color-primary-highlight: var(--ast-color-sphinx-design-primary);
+  --sd-color-primary-text: var(--ast-color-sphinx-design-background);
+  --sd-color-secondary: var(--ast-color-sphinx-design-background);
+  --sd-color-secondary-text: #141414;
+  --sd-color-secondary-highlight: var(--ast-color-hover-card-background);
+  --sd-color-secondary-bg: var(--ast-color-sphinx-design-background);
+  --ast-sd-bg-secondary-hover: var(--ast-color-hover-card-background);
+  --ast-background-tab-focus: #f5f5f5;
```

Comment:
Tab bg should always be transparent unless it's hovered (#ECECEC) or pressed (#D9D9D9). The attached image shows the latest component design. Added a pressed state to the original design.
![Tabs](https://github.com/user-attachments/assets/f1dbea24-4dbc-47e7-b108-8fbd49a1da33)